bluepyefe/reader.py

```diff
@@ -199,3 +200,90 @@ def nwb_reader(in_data):
         data = reader.read()
 
     return data
+
+
+def csv_lccr_reader(in_data):
+    """Reader to read .txt (csv_lccr)
+    Args:
+        in_data (dict): of the format
+        {
+            'filepath': "./XXX.txt",
+            'dt': 1
+            'ton': 2000,
+            'toff': 2500,
+            'ljp': 14.0,
+            'amplitudes': [0, 10, 20, 30, 40, 50, 60, 70, 80, 90],
+            'hypamp': -0.2
+            'remove_last_100ms': True,
+            'v_unit': 'mV',
+            't_unit': 'ms',
+            'i_unit': 'nA'
+        }
+    """
+    _check_metadata(
+        in_data,
+        csv_lccr_reader.__name__,
+        ["filepath", "dt", "amplitudes", "v_unit", "t_unit", "i_unit", "ton", "toff", "hypamp"],
+    )
+
+    data = []
+
+    fln = os.path.join(in_data['filepath'])
+    if not os.path.isfile(fln):
+        raise FileNotFoundError(
+            f"Please provide a string with the filename of the CSV file; "
+            f"current path not found: {fln}"
+        )
+
+    dt = in_data['dt']
+    ton = in_data['ton']
+    toff = in_data['toff']
+    amplitudes = in_data['amplitudes']
+    hypamp = in_data['hypamp']
+
+    import csv
+    with open(fln, 'rt') as f:
+        reader = csv.reader(f, delimiter='\t')
+        columns = list(zip(*reader))
+        length = numpy.shape(columns)[1]
+
+        voltage = numpy.array([
+            [
+                float(string) if string not in ["-", ""] else 0
```

Comment:
Have you seen this case of empty item or `-` in data?

Reply:
I haven't come across this in the data provided so far. Wouldn't it be prudent to maintain this check?"

Reply:
Yes, makes sense. Alright, keep it like this